lib/global_uid/server.rb

```diff
@@ -45,13 +45,16 @@ def disconnect!
       @allocator = nil
     end
 
-    def create_uid_table!(name:, uid_type:, start_id:, storage_engine:)
+    def create_uid_table!(name:, uid_type: nil, start_id: nil)
+      uid_type ||= "bigint(21) UNSIGNED"
+      start_id ||= 1
```

Comment:
Why not put the default values into the method signature? The magic of keyword arguments!

Reply:
To guard against `nil` values, these are extracted from an options hash. I could have checked for the presence of the key before passing it through but this felt safer